### ISSUE TITLE
Prise en compte de l'entretien et réparations de la voiture

### ISSUE THREAD

**Issue by Clemog** [incubateur-ademe/nosgestesclimat#1965]
On dispose de la données macro pour le poste "Commerce et réparation d'automobiles et de motocycles par habitant"

https://nosgestesclimat.fr/documentation/empreinte-branche/G45

<img width="789" alt="image" src="https://github.com/incubateur-ademe/nosgestesclimat/assets/55186402/a66717c7-2fac-4e40-af6a-82c8736033ae">

En considérant la partie "Entretien et réparation de véhicules automobiles", on pourrait ajouter une base pour l'empreinte de la voiture pour tous les propriétaires

**Comment by JuliePouliquen** [comment]
Cela va dans le sens également des réflexions concernant les maisons : faut-il prendre en compte la date de construction, ou de la dernière rénovation ?
Bon, la question ici est un peu différente parce que l'entretien est nécessaire et ne va pas forcément rallonger la durée de vie de l'automobile ad vitam eternam. Ni améliorer sa consommation ou quoique ce soit...
Mais cela rentre dans les réflexions "entretien, réparation des biens très durables (>10 ans)"

**Comment by Clemog** [comment]
Ici j'ai plus en tête le côté "hybridation du modèle" permis via cette branche économique

**Comment by Benjamin-Boisserie-ABC** [comment]
On a fait cette hybridation pour consolider les émissions manquantes post-enquête @Clemog @JuliePouliquen.

On s'est appuyé sur l'estimation GES du SDES (v2018) et sur les dépenses des ménages pour appliquer, par décile, un forfait d'émissions liés aux services d’entretien et réparation des véhicules personnels. 

Plus spécifiquement : 
 - les dépenses des ménages par décile en "0723 - Services d’entretien et réparation des véhicules personnels" ont été utilisées pour établir un facteur correctif par décile vis à vis de la moyenne des dépenses tous déciles confondus. Cf. [Excel TM106](https://www.insee.fr/fr/statistiques/4648319?sommaire=4648339) ligne 184.
 - ce facteur correctif est multiplié par les émissions de GES correspondant à l'entretien et réparation des véhicules (45.2 ; 45.3 et 45.4) estimées en :
   - utilisant le ratio de CA des branches (G_45.2+G_45.3+G_45.4)/G_45 pour estimer les GES (on prend en compte G.45.3 car on approxime une composante "distribution" des pièces nécessaires à la réparation. On prend en compte G_45.4 pour l'entretien des moto (même si le commerce est intégré dans ce code).
   - les émissions de GES liées aux ménages sur ce code G.45 (qui sont la somme de "emGES_dom_dtot_typ_FR18" qui représente les émissions associées à la demande intérieure produite, par produit et de "emGES_imp_hWO_dtot_typ_FR18" qui représente les "émissions associées à la demande intérieure importée, par produit"). Cf.
 
[V4a_FR_empreinte_GES_2018.xlsx](https://github.com/incubateur-ademe/nosgestesclimat/files/12078422/V4a_FR_empreinte_GES_2018.xlsx)

Pour intégration dans le modèle NGC se plusieurs problématiques spécifiques : 
- il serait préférable de séparer auto et moto (on a tous regroupé avec le Citepa au regard de l'urgence du redressement à faire)
- pas besoin d'une répartition par décile, je pense qu'on peut appliquer un forfait moyen qui compléterait notre FE construction pour qu'il se voit appliqué aux proprios d'auto ou moto
- à voir si on prend en compte le G_45.3 ? En théorie, notre FE construction ne prend pas en compte le fonctionnement des  et centre auto de réparation/entretien et même des concessionnaires (G45.1) qu'un automobiliste utilise. **Les prendre en compte est un parti pris méthodologique différent que ce que l'on faisait jusqu'à présent car cela consiste en quelque sorte à se caler sur le SDES. Je propose que pour le moment on prennent uniquement G.45.2 et G_45.4 et qu'on voit ce qu'il ressortira du GT Empreinte Carbone Individuelle lorsqu'on discutera de l'articulation entre approche micro/macro.**
- se pose la question des données GES. Ici on utilise les données 2018 décomposées différemment que les données 2017 dont on disposait jusque là. Cela permet de prendre en compte uniquement la part du code G_45 liée à la conso par les ménages. Dans un soucis de cohérence, il faudrait mettre à jour les services sociétaux avec les données 2018 (hors modifications éventuelles de l'approche services sociétaux liées aux recommandations du GT).
 
 Pour info, la nomenclature des dépenses des ménages COICOP est accessible [ici](https://www.insee.fr/fr/information/2493507) ... et n'est pas structuré de la même manière que la nomenclature NACE.

**Comment by JuliePouliquen** [comment]
Une question non néglieable quand on parle d'ajouter l'entretien, c'est : l'ajoutons-nous de manière égale pour les propriétaires de véhicules thermiques et hybrides ?
L'entretien des véhicules électriques est [moins fréquent et coûteux](https://www.estrepublicain.fr/magazine-automobile/2023/08/22/entretien-d-une-voiture-electrique-moins-cher-qu-un-vehicule-thermique) que celui des véhicules thermiques.

Peut-être à intégrer dans un second temps ?

**Comment by Clemog** [comment]
> * se pose la question des données GES. Ici on utilise les données 2018 décomposées différemment que les données 2017 dont on disposait jusque là. Cela permet de prendre en compte uniquement la part du code G_45 liée à la conso par les ménages. Dans un soucis de cohérence, il faudrait mettre à jour les services sociétaux avec les données 2018 (hors modifications éventuelles de l'approche services sociétaux liées aux recommandations du GT).

Yes carrément, on essaie de traiter https://github.com/incubateur-ademe/nosgestesclimat/issues/2057 au plus vite ;)
